packages/next/client/components/react-dev-overlay/internal/helpers/use-error-handler.ts

```diff
@@ -23,7 +23,10 @@ function isHydrationError(error: Error): boolean {
 }
 
 try {
-  Error.stackTraceLimit = 50
+  if (typeof window !== 'undefined') {
```

Comment:
Can you wrap the `if` around the `try` instead.

Reply:
Whoops, fixed 👍